bulk-upload-resources-python-example/example_config.txt

```diff
@@ -0,0 +1,6 @@
+[domain]
+data-path = https://data-staging.waylay.io
+resource-path = https://staging.waylay.io
+client-key = d596e2844bc0ee9500070d37
+client-secret = 6/Wr0XVnT+yvODLlRxEbpvC9SyyhDzaN
```

Comment:
Please don't commit API keys, this is a public repo. Remove these and make sure this commit is not accessible anymore by force pushing a reset.

Reply:
Or you can revoke your key here and ask for a new one: https://staging.waylay.io/#/profile/apikeys